src/parser.js

```diff
@@ -30,10 +30,55 @@ function parseToHtml(content, filename) {
     </html>`;
 }
 
+function mdParseToHtml(content, filename) {
+  let title = parseFileName(filename);
+
+  // to remove \r\n at the end of content
+  let str = content.slice(-1).toString();
+  str = str.slice(0, -2);
+  content.pop();
+  content.push(str);
+
+  // to convert md lines to html tags
+  const htmlContent = content
+    .map((line) => {
+      if (line.startsWith("# ")) {
+        var slicedLine = line.slice(2);
+        return `<h1>${slicedLine}</h1>`;
+      } else if (line.startsWith("## ")) {
+        var slicedLine = line.slice(3);
+        return `<h2>${slicedLine}</h2>`;
+      } else if (line.startsWith("### ")) {
+        var slicedLine = line.slice(3);
+        return `<h3>${slicedLine}</h3>`;
+      } else if (line.startsWith("_") && line.endsWith("_")) {
+        var slicedLine = line.slice(1);
+        slicedLine = slicedLine.slice(0, -1);
+        return `<i>${slicedLine}</i>`;
+      } else {
+        return `<p>${line}</p>`;
+      }
+    })
```

Comment:
I would like you to refer from using ``var`` since it's outdated just use a ``let``  and declare it before the line 42.

Is there a way that you can change this part ``line 54``  your are assuming that italics takes the entire line, could you implement it the way that would allow for something like this `` hello _italic_  hello``. I think this regex would help``(\*|_)(.*?)\1``

Reply:
Used 'let' instead of 'var' variable and added regex to find lines in italics as you mentioned. 

Please have a look and let me know if it's okay to merge. Thanks